internal/platform/marvell-dpu.go

```diff
@@ -28,9 +28,7 @@ func (d *MarvellDetector) Name() string {
 	return d.name
 }
 
-// IsDPU checks if the PCI device Attached to the host is a Marvell DPU
-// It returns true if device has Marvell DPU
```

Comment:
still checks if it's a marvell dpu. I presume you removed the incorrect comment after realizing that this returns true when it runs ON the dpu?

Reply:
Ok